src/index.ts

```diff
@@ -13,7 +13,7 @@ declare global {
 
 const appInit = function (
   settings: AppConfigInterface,
-  options: OptionsInterface = {},
```

Comment:
I think providing the default object as a fallback helps the `applyDefaults` function later on. I think it would error if this value is undefined.